google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -898,10 +898,6 @@ def default_acl
         # @see https://cloud.google.com/storage/docs/json_api/v1/buckets/setIamPolicy
         #   Buckets: setIamPolicy
         #
-        # @param [Boolean] force Force load the latest policy when `true`.
```

Comment:
#fwiw Sorry we didn't point this out with the initial API!  Of the GA interfaces, this was one of the last added and we *just* wrote samples with it.

Due to GA, `force` should be backwards compatible.  But I would be very pleased to see an additive `reload` named param in addition to a `#reload!` method

If we add `#reload!` but the parameter to reload when calling `Bucket#policy` is `force`, then that becomes an inconsistency.

Vote: Deprecate `force:` in favor of `reload:`.  Implementation something like `fetch_from_api! if reload || force`